Create Blog Post “2021-deaths-in-custody-decline-from-previous-year” (#802)
texas-justice-initiative/website-nextjs: content/blog/posts/2021-deaths-in-custody-decline-from-previous-year.md

---
title: 2021 Deaths in Custody decline from previous year
date: 2022-04-03
topics:
  - law enforcement
  - data oversight
authors:
  - Eva Ruth Moravec
---
The good news*: the number of people who died in the custody of Texas law enforcement went down in 2021 after peaking the previous year.

The bad news: 2021's total — 904 deaths, according to reports filed with the state of Texas through Feb. 1, 2022 — was still higher than every other year but 2020.

> Deaths in custody of law enforcement, including anything from a deadly shooting by a law enforcement officer or an overdose in the back of a patrol car to a death by suicide or medical cause in a prison cell, must be reported to the state of Texas within 30 days of occurring. Based on those reports, which TJI tracks every month, there were an average of 705 Texas deaths in custody from 2010 to 2020. Then, due undoubtedly in part to the 2019 coronavirus, deaths in custody rose almost 40% higher than the 10-year average before dropping slightly in 2021. 

Reviewer: I can't figure out how to do the in-line text editing so just leaving a comment. My recommended edit to the end of this sentence is:

deaths in custody rose almost 40% higher **in 2020** than the **prior** 10-year average before dropping slightly in 2021.

Also, do you mean to say the average is from 2010-2019 rather than including 2020 in that number?

Author: incorporated

> Deaths in custody of law enforcement, including anything from a deadly shooting by a law enforcement officer or an overdose in the back of a patrol car to a death by suicide or medical cause in a prison cell, must be reported to the state of Texas within 30 days of occurring. Based on those reports, which TJI tracks every month, there were an average of 705 Texas deaths in custody from 2010 to 2020. Then, due undoubtedly in part to the 2019 coronavirus, deaths in custody rose almost 40% higher than the 10-year average before dropping slightly in 2021. 

Reviewer: ```suggestion
Deaths in custody of law enforcement, including anything from a deadly shooting by a law enforcement officer or an overdose in the back of a patrol car to a death by suicide or medical cause in a prison cell, must be reported to the state of Texas within 30 days of occurring. Based on those reports, which TJI tracks every month, there were an average of 705 Texas deaths in custody from 2010 to 2019. Then, due undoubtedly in part to the 2019 coronavirus, deaths in custody rose almost 40% higher than the 10-year average before dropping slightly in 2021. 
```

Reviewer: this change is assuming the math has been updated. I dont remember what the numbers used to be

> Deaths in custody of law enforcement, including anything from a deadly shooting by a law enforcement officer or an overdose in the back of a patrol car to a death by suicide or medical cause in a prison cell, must be reported to the state of Texas within 30 days of occurring. Based on those reports, which TJI tracks every month, there were an average of 705 Texas deaths in custody from 2010 to 2019. Then, due undoubtedly in part to the 2019 coronavirus, deaths in custody rose almost 40% higher than the 10-year average before dropping slightly in 2021. 

Author: ```suggestion
Deaths in custody of law enforcement, including anything from a deadly shooting by a law enforcement officer or an overdose in the back of a patrol car to a death by suicide or medical cause in a prison cell, must be reported to the state of Texas within 30 days of occurring. Based on those reports, which TJI tracks every month, there were an average of 677 Texas deaths in custody from 2010 to 2019. Then, due undoubtedly in part to the 2019 coronavirus, deaths in custody rose nearly 45% higher than the prior 10-year average before dropping slightly in 2021. 
```

> The leading manner of death in 2021 continues to be of causes deemed “natural,” though the percentage of deaths by natural causes was fewer than in 2020 and for 2010 - 2020. The percentages of deaths due to “other” causes and “accidental” causes were slightly higher in 2021 than in 2020 and for 2010 - 2020, while deaths due to “homicide” were slightly lower. As in previous years, the highest concentration of custodial deaths in Texas took place among individuals over the age of 60. 

Reviewer: ```suggestion
The leading manner of death in 2021 continues to be of causes deemed “natural,” though the percentage of deaths by natural causes was fewer than in 2020 and for 2010 - 2019. The percentages of deaths due to “other” causes and “accidental” causes were slightly higher in 2021 than in 2020 and for 2010 - 2019, while deaths due to “homicide” were slightly lower. As in previous years, the highest concentration of custodial deaths in Texas took place among individuals over the age of 60. 
```

Reviewer: again this edit assumes the numbers have been updated

> The leading manner of death in 2021 continues to be of causes deemed “natural,” though the percentage of deaths by natural causes was fewer than in 2020 and for 2010 - 2020. The percentages of deaths due to “other” causes and “accidental” causes were slightly higher in 2021 than in 2020 and for 2010 - 2020, while deaths due to “homicide” were slightly lower. As in previous years, the highest concentration of custodial deaths in Texas took place among individuals over the age of 60. 

Author: thanks!

> *\* Good/bad news labels based on assumption that reducing the number of deaths that occur in the custody of Texas law enforcement is a positive thing*

Reviewer: lol
